pulsar-client-tools/src/main/java/org/apache/pulsar/shell/PulsarShell.java

```diff
@@ -567,6 +602,14 @@ private static boolean isQuitCommand(String line) {
         return line.equalsIgnoreCase("quit") || line.equalsIgnoreCase("exit");
     }
 
+    private static boolean isExitModeCommand(String line) {
+        return line.equalsIgnoreCase("exitmode");
+    }
```

Comment:
there shouldn't be a separate `exitmode` command. Simple `exit` or pressing CTRL-D should do that.